networkx/algorithms/broadcasting.py

```diff
@@ -37,27 +36,30 @@ def _get_broadcast_centers(G, v, values, target):
 @not_implemented_for("multigraph")
 @nx._dispatchable
 def tree_broadcast_center(G):
-    """Return the Broadcast Center of the tree `G`.
+    """Return the broadcast center of the tree `G`.
 
-    The broadcast center of a graph G denotes the set of nodes having
-    minimum broadcast time [1]_. This is a linear algorithm for determining
-    the broadcast center of a tree with ``N`` nodes, as a by-product it also
-    determines the broadcast time from the broadcast center.
+    The broadcast center of a graph `G` denotes the set of nodes having
+    minimum broadcast time [1]_. This function implements a linear algorithm
+    for determining the broadcast center of a tree with `n` nodes, as a
```

Comment:
The single backticks is only supposed to be for parameters (or `:obj:` link targets), so this should still be double backticks for rst-style inline formatting.

```suggestion
    for determining the broadcast center of a tree with ``n`` nodes, as a
```